cdn/common/monitor.sh

```
@@ -0,0 +1,128 @@
+# Environment variables:
+#   MONITOR_WEBHOOK           - Discord webhook URL
+#   MONITOR_MEMORY_THRESHOLD  - Alert when available memory drops below this % (default: 20)
+#   MONITOR_HEALTH_DOMAIN     - Domain name for health checks (default: cdn.moq.dev)
+#   MONITOR_HEALTH_JWT        - Path to subscriber JWT (default: /var/lib/moq/demo-sub.jwt)
+#   MONITOR_HEALTH_NODES      - Space-separated list of nodes (default: "usc euc sea")
+
+WEBHOOK="${MONITOR_WEBHOOK:-}"
+MEMORY_THRESHOLD="${MONITOR_MEMORY_THRESHOLD:-20}"
+HEALTH_DOMAIN="${MONITOR_HEALTH_DOMAIN:-cdn.moq.dev}"
+HEALTH_JWT_FILE="${MONITOR_HEALTH_JWT:-/var/lib/moq/demo-sub.jwt}"
+HEALTH_NODES=(${MONITOR_HEALTH_NODES:-usc euc sea})
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)